src/modelfree/score_agent.py

```diff
@@ -37,10 +32,11 @@ def announce_winner(sim_stream):
                 yield game_outcome(info)
 
 
-def get_empirical_score(_run, env, agents, episodes, render=False):
+@score_ex.capture
+def get_empirical_score(venv, agents, episodes, render, record_traj, _run):
     """Computes number of wins for each agent and ties.
 
-    :param env: (gym.Env) environment
+    :param venv: (VecEnv) vector environment
```

Comment:
I know this probably doesn't make sense to change for this PR, but I do wish our shortening for Vector Env didn't hash-collide with the canonical shortening of (python) Virtual Environment

Reply:
This is unfortunately standard in Baselines and elsewhere.